core-backend/sandbox/src/funcs.rs

```diff
@@ -379,6 +380,25 @@ where
         })
     }
 
+    pub fn random(ctx: &mut Runtime<E>, args: &[Value]) -> SyscallOutput {
```

Comment:
consider fail syscall on exceed len of subject?

Reply:
It fails in backend with `RuntimeBuffer::try_new_default()`